wolfssl-sys/patches/use-wolfssl-mlkem-kyber.patch

```diff
@@ -0,0 +1,3325 @@
+diff --git a/IDE/STM32Cube/wolfssl_example.c b/IDE/STM32Cube/wolfssl_example.c
+index 342e8ee9d0..be6195a60a 100644
+--- a/IDE/STM32Cube/wolfssl_example.c
++++ b/IDE/STM32Cube/wolfssl_example.c
```

Comment:
You should preferably have done something closer to

```
git format-patch HEAD~<Number_of_commits>
```

This will split the commits into multiple files so its clear what each patch does and it gives its originating commit.